### PR TITLE
feat(antrea): better docs for encyrption

### PR DESCRIPTION
**Pull request by DanStough**

### CODE REVIEW

**Review by ajp-io — DISMISSED** [review]

**Comment by ajp-io** [comment]
@replicatedhq/replicated-docs For your reivew.

File: src/markdown-pages/add-ons/antrea.md [hunk 1]
```diff
@@ -8,15 +8,20 @@ addOn: "antrea"
 ---
 
 [Antrea](https://antrea.io/) implements the Container Network Interface (CNI) to enable pod networking in a Kubernetes cluster.
-It also functions as a NetworkPolicy controller to optionallly enforce security at the network layer.
+It also functions as a NetworkPolicy controller to optionally enforce security at the network layer.
 Antrea is implemented with Open vSwitch and IPSec.
 
+By default, Antrea [encrypts traffic](https://antrea.io/docs/v1.4.0/docs/traffic-encryption/) between nodes. 
+kURL does not install the necessary kernel modules to enable traffic encryption.
+An installer will block if encryption is enabled and the host does not have the required `wireguard` module installed.
```
Comment (samantha-replicated) [inline src/markdown-pages/add-ons/antrea.md:16]:
```suggestion
An installer is blocked if encryption is enabled and the host does not have the required `wireguard` module installed.
```

File: src/markdown-pages/add-ons/antrea.md [hunk 2]
```diff
@@ -8,15 +8,20 @@ addOn: "antrea"
...
+Disabling encryption by setting `isEncryptionDisabled` to `true` prevents the need to manually install `wireguard`.
```
Comment (samantha-replicated) [inline src/markdown-pages/add-ons/antrea.md:17]:
```suggestion
If you do not want to install `wireguard` manually, you can disable encryption by setting `isEncryptionDisabled` to `true`. 
```

**Review by samantha-replicated — CHANGES_REQUESTED** [review]
Hi @DanStough. I added a couple edits. Just wanted you to review them first before committing to ensure I didn't impact the technical accuracy.

**Comment by ajp-io** [comment]
> Hi @DanStough. I added a couple edits. Just wanted you to review them first before committing to ensure I didn't impact the technical accuracy.

Thanks, those edits look fine to me. I committed them.

**Review by samantha-replicated — APPROVED** [review]

**Comment by samantha-replicated** [comment]
Approved. You can merge when you're ready :)

**Pull request merged**